src/material/dialog/dialog.spec.ts

```diff
@@ -1733,6 +1733,18 @@ describe('MatDialog', () => {
           .withContext('Expected the aria-labelledby to match the title id.')
           .toBe(title.id);
       }));
+
+      it('should add mat-dialog-actions-align-* class according to given [align] input in [mat-dialog-actions]', () => {
```

Comment:
Change to `it('should add correct css class according to given [align] input in [mat-dialog-actions]', () => {`